test/test_utils.py

```diff
@@ -115,8 +138,31 @@ def test_draw_segmentation_masks_colors(self):
 
     def test_draw_segmentation_masks_no_colors(self):
         img = torch.full((3, 20, 20), 255, dtype=torch.uint8)
-        # TODO
-        masks = torch.tensor([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12])
+        masks = torch.tensor(
+            [
+                [
+                    [-2.2799, -2.2799, -2.2799, -2.2799, -2.2799],
+                    [5.0914, 5.0914, 5.0914, 5.0914, 5.0914],
+                    [-2.2799, -2.2799, -2.2799, -2.2799, -2.2799],
+                    [-2.2799, -2.2799, -2.2799, -2.2799, -2.2799],
+                    [-2.2799, -2.2799, -2.2799, -2.2799, -2.2799]
+                ],
+
+                [
+                    [5.0914, 5.0914, 5.0914, 5.0914, 5.0914],
+                    [-2.2799, -2.2799, -2.2799, -2.2799, -2.2799],
+                    [5.0914, 5.0914, 5.0914, 5.0914, 5.0914],
+                    [5.0914, 5.0914, 5.0914, 5.0914, 5.0914],
+                    [-1.4541, -1.4541, -1.4541, -1.4541, -1.4541]
+                ],
+                [
+                    [-1.4541, -1.4541, -1.4541, -1.4541, -1.4541],
+                    [-1.4541, -1.4541, -1.4541, -1.4541, -1.4541],
+                    [-1.4541, -1.4541, -1.4541, -1.4541, -1.4541],
+                    [-1.4541, -1.4541, -1.4541, -1.4541, -1.4541],
+                    [5.0914, 5.0914, 5.0914, 5.0914, 5.0914],
+                ]
+            ], dtype=torch.float)
```

Comment:
Nit: I wonder if we could refactor the 2 tests (perhaps merge them?) to avoid having duplicate code here?

Reply:
Not sure if we can merge, coz both test slightly different functionality (easier to debug later on) + in place modification isn't tested. I moved the declaration globally to avoid duplication.

Reply:
Sounds good to me. Thanks!